packages/jetbrains-plugin/src/test/kotlin/com/mongodb/jbplugin/fixtures/components/idea/IdeaFrame.kt

```diff
@@ -0,0 +1,93 @@
+/**
+ * Represents the main frame of IDEA. The one you see when you open a project, the whole window.
+ * Reference code taken from:
+ * https://github.com/JetBrains/intellij-ui-test-robot/blob/master/ui-test-example/src/test/kotlin/org/intellij/examples/simple/plugin/pages/IdeaFrame.kt
+ */
+
+package com.mongodb.jbplugin.fixtures.components.idea
+
+import com.intellij.remoterobot.RemoteRobot
+import com.intellij.remoterobot.data.RemoteComponent
+import com.intellij.remoterobot.fixtures.*
+import com.mongodb.jbplugin.fixtures.findVisible
+
+/**
+ * Fixture that represents the frame itself. You can add more functions here if you want to interact
+ * with the frame.
+ *
+ * @param remoteRobot
+ * @param remoteComponent
+ */
+@FixtureName("Idea frame")
+@DefaultXpath("IdeFrameImpl type", "//div[@class='IdeFrameImpl']")
+class IdeaFrame(
+    remoteRobot: RemoteRobot,
+    remoteComponent: RemoteComponent,
+) : CommonContainerFixture(remoteRobot, remoteComponent) {
+    fun openFile(path: String) {
+        this.closeAllFiles()
+
+        runJs(
+            """
+            importPackage(com.intellij.openapi.fileEditor)
+            importPackage(com.intellij.openapi.vfs)
+            importPackage(com.intellij.openapi.wm.impl)
+            importClass(com.intellij.openapi.application.ApplicationManager)
+            
+            const path = '$path'
```

Comment:
Should this be escaped?

Reply:
It's only for testing so I don't think it's that bad, but yeah I can encode it just in case we don't break anything during a test.